lib/functions.php

```diff
@@ -1,5 +1,15 @@
 <?php
 
+/**
+ * Check if the P6 installer plugin is active
+ *
+ * @return booelan
+ */
+function p7i_p6_installer_active() {
+    $activePlugins = get_option('active_plugins');
+    return in_array('pp-installer-prophoto6/pp-installer.php', $activePlugins);
```

Comment:
I don't think this would match p6 plugins in the wild, the `pp-installer-prophoto6` is not what the dir is named for our customers. I think there is a more safe way to check if the plugin is active, I forget the function name, but you can list out the active plugins (EDIT: use a combo of `get_plugins()` and `is_plugin_active()`).